src/App.js

```diff
@@ -7,7 +7,7 @@ import Quotes from './components/pages/Quotes';
 
 const App = () => (
   <Router>
-    <div className="App">
+    <main className="App">
```

Comment:
Good usage of HTML5 Semantic elements, Additionally I would like to suggest replacing the `<div>` tag with `id='root'` with the `<main>` tag.
I think it would be a better place to locate the `<main>` tag.